src/Microsoft.VisualStudio.ProjectSystem.Managed/Text/StringHash.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Diagnostics.Contracts;
+
+namespace Microsoft.VisualStudio.Text
```

Comment:
For new files, consider using file-scoped namespaces.

Reply:
The code was moved to BuildUpToDateCheck.ItemHashing.cs.

done.